packages/fxa-email-service/README.md

```diff
@@ -129,8 +127,8 @@ You'll need to set up some config
 with your AWS credentials.
 That can be with environment variables:
 
-* `FXA_EMAIL_AWS_KEYS_ACCESS`
-* `FXA_EMAIL_AWS_KEYS_SECRET`
+- `FXA_EMAIL_AWS_KEYS_ACCESS`
```

Comment:
Huh, why do some editors/prettier configs convert to `-`, others to `*`, mine always converts these back to `*`.